python/doc/examples/probabilistic_modeling/distributions/plot_pointConditionalDistribution.py

```diff
@@ -0,0 +1,255 @@
+view = otv.View(grid)
+
+
+# %%
+# Case 3: Trivariate distribution with continuous and discrete marginals
+# ----------------------------------------------------------------------
+#
+# Here, we consider the trivariate distribution of :math:`(X_0, X_1, X_2)` defined by:
+# .. math::
+#
+#   X_0 & \sim \cN(0, 1)
+#   X_1 & \sim \cP(1)
+#   X_2 & \sim \cU(-1, 1)
+#   \mbox{copula} & \sim \mbox{NormalCopula}(3)
+#
+# where the correlation matrix of the normal copula is such that:
+# .. math::
+#
+#   R = \left( \begin{array}{ccc}
+#       1 & 0.2 & 0.7 \\
+#       0.2 & 1 & -0.5 \\
+#       0.7 & -0.5 & 1
+#       \end{array} \right)
+#
+# We condition the marginal :math:`X_1` to be simultaneously equal to its quantiles of order
+# 0.05, 0.5 and 0.95. Then, we draw the resulting bivariate conditioned distribution of:
+# .. math::
+#
+#    (X_0,X_2) | X_1 = x_1
+#
+# We go through the same steps as before.
+coll_marg = [ot.Normal(), ot.Poisson(1.0), ot.Uniform()]
+R = ot.CorrelationMatrix(3)
+R[0, 1] = 0.2
+R[0, 2] = 0.7
+R[1, 2] = -0.5
+cop = ot.NormalCopula(R)
+dist_X = ot.JointDistribution(coll_marg, cop)
+
+# %%
+# We draw the distribution of :math:`(X_0, X_2)`
+g_X02 = dist_X.getMarginal([0, 2]).drawPDF([-4.0, -1.0], [4.0, 1.0], [256] * 2)
+contour = g_X02.getDrawable(0).getImplementation()
+contour.setIsFilled(True)
+contour.buildDefaultLevels(50)
+g_X02.setDrawable(contour, 0)
+g_X02.setTitle(r"$(X_0, X_2)$: iso-lines PDF")
+
+# We draw all the conditioned distributions :math:`(X_0, X_2)|X_1 = x_1s`.
+cond_value_list = [coll_marg[1].computeQuantile(q)[0] for q in q_list]
+graph_list = list()
+for index, cond_value in enumerate(cond_value_list):
+    cond_value = cond_value_list[index]
+    cond_dist = otexp.PointConditionalDistribution(dist_X, cond_indices, [cond_value])
+    g_cond = cond_dist.drawPDF([-4.0, -1.0], [4.0, 1.0], [256] * 2)
+    contour = g_cond.getDrawable(0).getImplementation()
+    contour.setIsFilled(True)
+    contour.buildDefaultLevels(50)
+    g_cond.setDrawable(contour, 0)
+    g_cond.setTitle(r"$(X_0, X_2)|X_1 = q($" + str(q_list[index]) + "): iso-lines PDF")
+    g_cond.setXTitle(r"$x_0$")
+    g_cond.setYTitle(r"$x_2$")
+    graph_list.append(g_cond)
+    print("Quantile of the cond value = ", q_list[index])
+    print("Range :\n", cond_dist.getRange())
+
+
+# %%
+# We gather both graphs into one grid.
+grid = ot.GridLayout(2, 2)
+grid.setGraph(0, 0, g_X02)
+grid.setGraph(0, 1, graph_list[0])
+grid.setGraph(1, 0, graph_list[1])
+grid.setGraph(1, 1, graph_list[2])
+view = otv.View(grid)
+
+# %%
+# Case 4: Extreme values conditioning
+# -----------------------------------
+#
+# Here, we show that it is possible to condition one marginal to a value which is outside its numerical range.
+# Be careful that the conditioning value still must be in the theoretical range of the distribution!
+#
+# We consider the bivariate  distribution of :math:`(X_0, X_1)` defined by:
+# .. math::
+#
+#   X_0 & \sim \cN(0, 1)
```

Comment:
There is no universal notation for each distribution: I suggest to explain what means the caligraphic N.